hack/verify-all.sh

```diff
@@ -10,9 +10,9 @@ source "${ROOT_PATH}/hack/lib/init.sh"
 go::verify_go_version
 
 bash "$ROOT_PATH/hack/verify-shfmt.sh"
-bash "$ROOT_PATH/hack/verify-vendor.sh"
+#bash "$ROOT_PATH/hack/verify-vendor.sh"
```

Comment:
The files here is from the previous general specification.

According to the current project, we can make some changes:

1. The vendor-related scripts are not needed and can be removed.
2. The helm-related scripts are not needed and can be removed.
3. The codegen-related scripts are not needed and can be removed. （ We are using `make generate`, and we may need to confirm the operation of this command.)
4. The crdgen-related scripts are not needed and can be removed. （ We are using `make manifests`, and we may need to confirm the operation of this command.)